docs/framework/migration-guide/application-compatibility.md

```diff
@@ -15,128 +15,81 @@ helpviewer_keywords:
+still work; however, when a change must be made to previous functionality (for
+performance, security, or a bug) this can cause compatibility problems in
+already written code. A distinction to understand with .NET compatibility is
+runtime vs retargeting issues.
+
+When developing an app the .NET Framework version that the app is targeting is
+specified either by specifying a target Framework in Visual Studio, specifying
+the target Framework in a project file, or applying a TargetFrameworkAttribute
+to the source code.
+
+When running on a newer version than what was targeted, the .NET Framework will
+use quirked behavior to mimic the older targeted version. In other words, the
+app will run on the newer version of the Framework, but act as if it's running
+on the older version. Many of the breaking changes between versions of the .NET
+Framework are mitigated through this quirking model.
+
+Runtime issues are those that arise when a new runtime is placed on a machine
+and the same binaries are run, but different behavior is seen. If a binary was
+compiled for .NET 4.0 it will run in .NET 4.0 compatibility mode on 4.5+. Many
+of the changes that affect 4.5 will not affect a binary compiled for 4.0. This
+is specific to the AppDomain and depends on the settings of the entry assembly.
+
+Retargeting issues are those that arise when an assembly that was targeting 4.0
+is now set to target 4.5. Now the assembly opts into the new features as well as
+potential breaking changes to old features. Again, this is dictated by the entry
+assembly, so the console app that uses the assembly, or the website that
+references the assembly.
+
+The .NET Compatibility Diagnostics are Roslyn-powered analyzers that help
+identify application compatibility issues between versions of the .NET
+Framework. This list contains all of the analyzers available, although only a
+subset will apply to any specific migration. The analyzers will determine which
+issues are applicable for the planned migration and will only surface those.
+
+Each issue includes the following information:
+
+-   The description of what has changed from a previous version.
+
+-   The suggestion is a description of how the change affects customers and whether any workarounds are available to preserve compatibility across versions.
+
+-   An assessment of how important the change is. Application compatibility issue are categorized as follows:
+
+    |   |   |
+    |---|---|
+    |Major|A significant change that affects a large number of apps or requires substantial modification of code.|
+    |Minor|A change that affects a small number of apps or that requires minor modification of code.|
+    |Edge case|A change that affects apps under very specific, uncommon scenarios.|
+    |Transparent|A change with no noticeable effect on the application's developer or user.|
+
+-   Version indicates when the change first appears in the framework. Some of the changes are reverted and that is indicated as well.
```

Comment:
"reverted" is likely to cause confusion. "Some of the changes are introduced in a particular version and reverted in a later version; that is indicated as well."